src/amber/cli/recipes/filters.cr

```diff
@@ -0,0 +1,54 @@
+# add a pluralize filter to liquid
+# this should be a temporary measure until pluralize is available as a filter
+# in the liquid shard
+
+require "inflector"
+
+module Liquid::Filters
+
+  FilterRegister.register "pluralize", Pluralize
```

Comment:
Given that this code is written as a shim until the liquid project gets this filter, will this code break when the liquid project is updated?

Reply:
no it will simply overwrite the existing filter